python/gui/MaaS/cbv/DatasetManagementView.py

```diff
@@ -0,0 +1,114 @@
+"""
+Defines a view that may be used to configure a MaaS request
+"""
+import asyncio
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+
+import dmod.communication as communication
+from dmod.core.meta_data import DataCategory, DataFormat
+
+import logging
+logger = logging.getLogger("gui_log")
+
+from .utils import extract_log_data
+from .AbstractDatasetView import AbstractDatasetView
+
+
+class DatasetManagementView(AbstractDatasetView):
```

Comment:
Thoughts on adding a `delete` method here to handle deleting a dataset? Or will that fall under the responsibility of `post`? Meaning, `post` would then likely handle creation, deletion, and modification.